vertx/public/dataviewer/src/PropertyViewModel.js

```diff
@@ -107,14 +111,18 @@ module.exports = function(tree, shouter, worker) {
         propertiesElem.show();
         $('.dataValueInput').hide();
         input.hide();
+        var aclModif = (aclOptions.value).replace('$p', 'P');
+        var testACL = (csync.acl)[aclModif];
```

Comment:
the reason this is happening is if I send a direct $publicCreate to the js SDK it will not accept it. It need to do `csync.acl.PublicCreate`